### PR TITLE
fix(item): fix label alignment in item with icon

### PR DESCRIPTION
**Pull request by liamdebeasi**
#### Short description of what this resolves:
This PR fixes an issue where labels with icons in an item are slightly misaligned.

#### Changes proposed in this pull request:

- Change md label margin-top to 10px
- Change line-height of label to 18px
-

**Ionic Version**:

**Fixes**: #16422

### CODE REVIEW

**Comment by brandyscarney** [comment]
So the label looks better in certain cases like with an icon:

screen shot 2019-02-14 at 1 33 47 pm

but in some other instances it looks a bit off to me:

screen shot 2019-02-14 at 1 33 26 pm
screen shot 2019-02-14 at 1 34 42 pm

Maybe we need to rethink this implementation?

**Comment by liamdebeasi** [comment]
In the non-icon images, were the labels fine before the change? If so, maybe those could be an icon-only rule?

**Comment by brandyscarney** [comment]
@liamdebeasi Yeah the images on the left are prior to this PR. See the screenshot check: https://github.com/ionic-team/ionic/pull/17489/checks?check_run_id=63468433

File: core/src/components/item/item.md.scss [hunk 1]
```diff
@@ -191,6 +191,8 @@
 
 ::slotted(ion-label) {
   @include margin($item-md-label-margin-top, $item-md-label-margin-end, $item-md-label-margin-bottom, $item-md-label-margin-start);
+  
+  line-height: 18px;
```
Comment (manucorporat) [inline core/src/components/item/item.md.scss]: can this be a relative value?

**Comment by brandyscarney** [comment]
Does this need to be re-created? Why is screenshot not running?

**Comment by liamdebeasi** [comment]
Closing as this does not properly address the issue.

**Pull request closed without merging**